docs/understand/enterprise/orchestration.md

```diff
@@ -0,0 +1,389 @@
+   1. Reply URL: lakefs-url/saml (e.g. https://lakefs.acme.com/saml)
+   1. Sign on URL: lakefs-url/sso/login-saml (e.g. https://lakefs.acme.com/sso/login-saml)
+   1. Relay State (Optional): /
+
+### Fluffy Configuration
+
+**Note:** Full Fluffy configuration can be found [here]({% link understand/enterprise/fluffy-configuration.md %})..
+{: .note }
+
+1. `auth.saml.idp_metadata_url`: Set from the Azure app created above _SAML configuration > “App Federation Metadata Url”_
+1. `auth.saml.external_user_id_claim_name`: The claim that represents the UserID.
+   The saim claim name must also be set in lakeFS config key `auth.cookie_auth_verification.external_user_id_claim_name`.
+   The example below uses http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name as the claim name.
+
+```yaml
+# fluffy configuration.yaml 
+
+# SSO address (i.e login with Azure) and healthcheck address /_health
+listen_address: :8000
+logging:
+  level: "INFO"
+  audit_log_level: "INFO"
+# everything under database equal to the config in lakeFS
+database:
+  type: postgres
+  postgres:
+    # same as lakeFS
+    connection_string: <postgres connection string> 
+auth:
+  # RBAC Service, default address is 0.0.0.0:9000, also lakeFS healcheck
+  serve_listen_address: ':9000'
+  encrypt:
+    # same as lakeFS
+    secret_key: shared-secrey-key 
+  logout_redirect_url: https://<lakefs-url>
+  post_login_redirect_url: https://<lakefs-url>
+  saml:
+    enabled: true
+    sp_root_url: https://<lakefs-url>
+    # generated SSL key, if not enforced on on the IdP level then can be anything as long as it a valid cert structure. 
+    sp_x509_key_path: '/etc/saml_certs/rsa_saml_private.cert'
+    sp_x509_cert_path: '/etc/saml_certs/rsa_saml_public.pem'
+    sp_sign_request: false
+    sp_signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
+    idp_metadata_url: https://login.microsoftonline.com/<...>/federationmetadata/2007-06/federationmetadata.xml?appid=<app-id>
+    external_user_id_claim_name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+    idp_skip_verify_tls_cert: true
+```
+
+### lakeFS Configuration
+
+```yaml
+# lakeFS configuration.yaml 
+
+database:
+  type: postgres
+  postgres:
+    # same as fluffy
+    connection_string: <postgres connection string> 
+auth:
+  encrypt:
+    # same as fluffy
+    secret_key: shared-secrey-key 
+  api:
+    # RBAC endpoint
+    endpoint: http://<fluffy>:9000/api/v1
+  cookie_auth_verification:
+    auth_source: saml
+    friendly_name_claim_name: displayName
+    external_user_id_claim_name: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+    default_initial_groups:
+      - "Developers"
+  ui_config:
+    login_url: "https://<lakefs-url>/sso/login-saml"
+    logout_url: "https://<lakefs-url>/sso/logout-saml"
+    rbac: internal
+    login_cookie_names:
+       - internal_auth_session
+       - saml_auth_session
+```
+
+  </div>
+  <div markdown="1" id="ldap">
```

Comment:
This seems like duplicate of [LDAP SSO](https://docs.lakefs.io/reference/security/sso.html#ldap), can we have one source of truth?

Reply:
Right, but it's in the Enterprise docs context. For the users who just want this to work, I think a copy-paste solution for them would be better than relying on another page that is the general LDAP case (For Cloud too).